internal/output/summary/data.go

```diff
@@ -294,8 +294,8 @@ func populateSummaryChecks(
 		getMetricValues(checksMetric.Sink, testRunDuration),
 	)
 
-	summaryGroup.Checks.Metrics.Fail.Values["passes"] = totalChecks - successChecks
-	summaryGroup.Checks.Metrics.Fail.Values["fails"] = successChecks
+	summaryGroup.Checks.Metrics.Fail.Values["passes"] = successChecks
+	summaryGroup.Checks.Metrics.Fail.Values["fails"] = totalChecks - successChecks
```

Comment:
Please, note that these lines:
- First and foremost, refer to checks metrics, not browser metrics, as reported in the issue #4751
- Second, and despite I know it can be confusing, `"passes"` and `"fails"` there are the identifiers used for values of `*metrics.RateSink`, and as you can see, it refers to: `summaryGroup.Checks.Metrics.Fail`. So, "passes" really correspond to: _"how many times checks failed"_ (note the difference with `summaryGroup.Checks.Metrics.Success`.